test/gerkin/tests/features/data_update_entries.feature

```diff
@@ -0,0 +1,15 @@
+Feature: API endpoint allowing users to update multiple records in the Digital Registries database.
```

Comment:
For this feature I would see three scenarios:
- request to update multiple records AND all of them exist
- request to update multiple records AND only some of them exist
- request to update multiple records AND none of them exist

Described scenarios are good but I would expect a little bit more details what's a valid/invalid payload